pkg/backup/compaction_processor.go

```diff
@@ -280,7 +305,7 @@ func (p *compactBackupsProcessor) processSpanEntries(
 				return errors.Wrap(err, "opening SSTs")
 			}
 
-			if err := compactSpanEntry(ctx, sstIter, sink); err != nil {
+			if err := compactSpanEntry(ctx, sstIter, sink, pacer); err != nil {
```

Comment:
`MakeSSTSinkKeyWriter` already has an input arg for a pacer which we are not using. Could you remove that arg in `MakeSSTSinkKeyWriter`? I think it makes sense to pass a pacer to `CompacSpanEntry`, as you have done here, as we're pacing outside of file sink code.

Reply:
Done!